### DISCUSSION TITLE
Shared `Header` component?

### DISCUSSION THREAD

**Discussion by oliverlloyd** [guardian/source#816] [Ideas]
## `Header`
How about we have a shared Header component?

![Screenshot 2021-05-12 at 13 59 26](https://user-images.githubusercontent.com/1336821/117978973-56220300-b32a-11eb-9c23-ae33a97f8d91.jpg)
![Screenshot 2021-05-12 at 13 58 41](https://user-images.githubusercontent.com/1336821/117978977-56ba9980-b32a-11eb-99e8-c9511ccfc31d.jpg)
![Screenshot 2021-05-12 at 13 58 27](https://user-images.githubusercontent.com/1336821/117978981-57533000-b32a-11eb-82d6-bd407ffd06ca.jpg)
![Screenshot 2021-05-12 at 13 58 11](https://user-images.githubusercontent.com/1336821/117978985-57ebc680-b32a-11eb-81f4-620375b48ded.jpg)
![Screenshot 2021-05-12 at 13 57 40](https://user-images.githubusercontent.com/1336821/117978987-57ebc680-b32a-11eb-9387-b1218b642d50.jpg)
![Screenshot 2021-05-12 at 13 57 26](https://user-images.githubusercontent.com/1336821/117978988-58845d00-b32a-11eb-991e-47ec3aee635f.jpg)

There's already been [some work](https://www.figma.com/file/V8u2r27TFdjAWNM3p84ypC/%E2%9C%AD-Headers-%26-Footers?node-id=0%3A1) in this space as well as meetings and discussions and from this I think it's fair to say there's general agreement that this is a good idea but there are still some  outstanding questions:

1. Should we use the Roundel the TheGuardian word mark? (or both depending on breakpoint?)
2. What height should the header be?
3. Does it need a title on the left?
4. Where should the MyAccount dropdown sit?
5. Where should links sit?
6. Do we want to try and encapsulate the main site header or just focus on admin use cases?
7. Should this component manage column logic (left and right margins at wider breakpoints) or stretch to it's parent and let the calling component manage this?

**Comment by dom-whooley** [comment]
As you say some of the work has been already done on this in the design system. Can we list urls here where the examples above have come from, that should help our investigation and decision making.

**Reply by oliverlloyd** [reply]
> Can we list urls here where the examples above have come from

I'm getting examples from these pages

https://manage.theguardian.com/help-centre
https://profile.theguardian.com/reset
https://manage.theguardian.com/billing
https://www.theguardian.com/about/history
https://www.theguardian.com/uk

**Reply by dom-whooley** [reply]
https://www.theguardian.com/email-newsletters?INTCMP=DOTCOM_NAV_NEWSLETTER_US

**Reply by oliverlloyd** [reply]
That looks like the immersive header
https://www.theguardian.com/news/2021/may/06/between-two-worlds-a-chinese-american-story

I wonder if that's maybe a different thing - it's more like the `Nav` but with a Roundel on the side. (`Nav` as in the Pillar nav that sits underneath the Header on most articles).

On the other hand, it _is_ a 'header' in these contexts  🤷

**Reply by dom-whooley** [reply]
Yeah, it looks the same as the immersive header. I posted it here to reference where these variations happen.
